input/i18n/en/my-account.yaml

```diff
@@ -21,3 +21,9 @@ personalDetails:
   welcomeBack: Welcome back
   welcomeDescription: for an even better customer service please provide your customer number
   personalDetailsTitle: Your Personal Details
+shippingAddress:
```

Comment:
I would put all the address book related text inside `addressBook:`, otherwise `shippingAddress` and especially `buttons` are likely to be duplicated in other areas of my account.